packages/backend-api/install/app/AppKernel.php.patch

```diff
@@ -0,0 +1,36 @@
+@@ -19,8 +19,10 @@
+             new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
+             new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
+             new FM\ElfinderBundle\FMElfinderBundle(),
++            new FOS\RestBundle\FOSRestBundle(),
+             new Fp\JsFormValidatorBundle\FpJsFormValidatorBundle(),
+             new Intaro\PostgresSearchBundle\IntaroPostgresSearchBundle(),
++            new JMS\SerializerBundle\JMSSerializerBundle(),
+             new JMS\TranslationBundle\JMSTranslationBundle(),
+             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
+             new Presta\SitemapBundle\PrestaSitemapBundle(),
+@@ -33,6 +35,7 @@
+             new Shopsys\ProductFeed\HeurekaDeliveryBundle\ShopsysProductFeedHeurekaDeliveryBundle(),
+             new Shopsys\ProductFeed\ZboziBundle\ShopsysProductFeedZboziBundle(),
+             new Shopsys\ProductFeed\GoogleBundle\ShopsysProductFeedGoogleBundle(),
++            new Shopsys\BackendApiBundle\ShopsysBackendApiBundle(),
+             new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
+             new Snc\RedisBundle\SncRedisBundle(),
+             new Symfony\Bundle\AsseticBundle\AsseticBundle(),
+@@ -42,6 +45,7 @@
+             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
+             new Symfony\Bundle\TwigBundle\TwigBundle(),
+             new Symfony\Cmf\Bundle\RoutingBundle\CmfRoutingBundle(),
++            new Trikoder\Bundle\OAuth2Bundle\TrikoderOAuth2Bundle(),
+             new VasekPurchart\ConsoleErrorsBundle\ConsoleErrorsBundle(),
+             new FOS\CKEditorBundle\FOSCKEditorBundle(), // has to be loaded after FrameworkBundle and TwigBundle
+             new Joschi127\DoctrineEntityOverrideBundle\Joschi127DoctrineEntityOverrideBundle(),
+@@ -104,5 +108,9 @@
```

Comment:
```suggestion
@@ -104,4 +108,8 @@
```

Before it left me untracked file `AppKernel.php.orig` on uninstallation. With the suggestion applied everything works without problem. 
(Maybe something with the empty line at the end)

Reply:
applied manually. Both versions are ok in docker